src/main/java/de/unistuttgart/t2/uibackend/exceptions/CartInteractionFailedException.java

```diff
@@ -2,10 +2,12 @@
 
 /**
  * Indicates that interaction with the cart service failed.
- * 
+ *
  * @author maumau
  */
-public class CartInteractionFailedException extends Exception {
+public final class CartInteractionFailedException extends Exception {
+
+    private static final long serialVersionUID = 1L;
```

Comment:
I'm not sure if a 1 is better than nothing

Reply:
Do you know why `serialVersionUID`s are needed for Exceptions?
Every serializable class needs them, so that the JVM can determine if it can deserialize a serialized object.
Exceptions are serializable.
Even if you don't use serialization, Java/ Eclipse will complain otherwise.
Normally, you would change/ **increment** them, when the source classes change.

Reply:
I know for what these are.  
However, setting them manually also requires you to update them, which a lot of people forget. Therefore, I typically prefer the automatically generated ones, as you need really bad luck that you have a conflict there. However, the Internet seems split on this, so I think it probably does not matter that much.  
When setting it manually, I prefer the generated one because it keeps compatability with the previous version of the program, however, again, I don't think it's that important here

Reply:
Okay, so what's your preference now? Generated or Counter?

Reply:
Let's just leave it like this, it's not so important
Use your own preference